apps/src/templates/sectionProgress/VirtualizedDetailView.jsx

```diff
@@ -146,7 +147,9 @@ class VirtualizedDetailView extends Component {
             {scriptData.stages[stageIdIndex].levels.map((level, i) =>
               <FontAwesome
                 icon={getIconForLevel(level)}
-                style={progressStyles.icon}
+                style={
+                  level.kind === "unplugged" ? progressStyles.unpluggedIcon : progressStyles.icon
```

Comment:
Not for this PR, but wanted to make a note for the future. Ram pointed out we should factor out some section data processing from `progressRedux`, and then we can instead use `.isUnplugged` and our string will only need to be defined in one spot.
Current progressRedux functions combine progress with the section data we want, so we'd have to change these functions.
https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/progressRedux.js#L408-L419

We've used `level.kind === "unplugged"` in a couple other places in section progress so it's not a big deal and is non-blocking.